setup.py

```diff
@@ -57,7 +57,7 @@
         "spacy>=2.1.0,<2.4",
         "numpy",
         "tensorboardX>=1.2",
-        "boto3",
+        "boto3>=1.14",
```

Comment:
At this point I think we should also pin this below `2.0`, since dependabot will automatically create a PR to update this requirement when 2.0 comes out.

```suggestion
        "boto3>=1.14,<2.0",
```